lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -514,6 +515,13 @@ class BuildOpGroups : public calyx::FuncOpPartialLoweringPattern {
                                /*subtract=*/1);
       }
       rewriter.create<calyx::AssignOp>(loc, opFOp.getSubOp(), subOp);
+    } else if (isa<calyx::DivSqrtOpIEEE754>(opPipe)) {
+      auto opFOp = cast<calyx::DivSqrtOpIEEE754>(opPipe);
+      hw::ConstantOp sqrtOp;
+      bool isSqrt = !isa<arith::DivFOp>(op);
+      sqrtOp =
+          createConstant(loc, rewriter, getComponent(), /*width=*/1, isSqrt);
```

Comment:
Sorry this is probably from my code suggestion; we can probably just assign on the same line now.
```suggestion
      bool isSqrt = !isa<arith::DivFOp>(op);
      hw::ConstantOp sqrtOp sqrtOp =
          createConstant(loc, rewriter, getComponent(), /*width=*/1, isSqrt);
```